$MirOS: ports/emulators/qemu/patches/patch-hw_fmopl_c,v 1.1 2005/11/15 12:25:08 tg Exp $
--- hw/fmopl.c.orig	Tue Nov 15 12:00:25 2005
+++ hw/fmopl.c	Tue Nov 15 12:00:48 2005
@@ -1343,8 +1343,9 @@ unsigned char OPLRead(FM_OPL *OPL,int a)
 		{
 			if(OPL->keyboardhandler_r)
 				return OPL->keyboardhandler_r(OPL->keyboard_param);
-			else
+			else {
 				LOG(LOG_WAR,("OPL:read unmapped KEYBOARD port\n"));
+			}
 		}
 		return 0;
 #if 0
@@ -1356,8 +1357,9 @@ unsigned char OPLRead(FM_OPL *OPL,int a)
 		{
 			if(OPL->porthandler_r)
 				return OPL->porthandler_r(OPL->port_param);
-			else
+			else {
 				LOG(LOG_WAR,("OPL:read unmapped I/O port\n"));
+			}
 		}
 		return 0;
 	case 0x1a: /* PCM-DATA    */
